server/repository/src/db_diesel/item_row.rs

```diff
@@ -153,7 +153,7 @@ impl<'a> ItemRowRepository<'a> {
         Ok(result)
     }
 
-    pub fn find_many_by_id(&self, ids: &[String]) -> Result<Vec<ItemRow>, RepositoryError> {
+    pub fn find_many_by_id(&self, ids: &Vec<String>) -> Result<Vec<ItemRow>, RepositoryError> {
```

Comment:
Nothing else was using this surprisingly!
Seems much easier to create a vec rather than an array when we don't know the size...